CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 23.2.0 [#675](https://github.com/openfisca/openfisca-core/pull/675)
+
+- Add performance testing for the test-runner 
+  - `pip install -e .[performance]` installs the necessary dependencies
```

Comment:
Having said that, this makes me wonder if this performance script would not be better located in `openfisca-france`. 

I understand why we thought about putting it there, and I totally agree that France tells us a lot about Core performance, as it's the only model where we push the limits in term of model complexity.

However:

- The fact that we have to go out of our way to access these files smells like we are introducing a dependencies that is not very natural.
- We are not _really_ testing the performance of the test runner. The test runner is only a small cog in the machinery. We are testing the performance of many small (in term of number of people) OpenFisca-France simulations by using the test runner. So it's not clear, when looking at the performance result, what part is due to OpenFisca-Core in general, and what part is due to implementation choices in OpenFisca-France.

Reply:
Would love to see that land in Core if it was possible to add the measurement part in it, then letting each country use the tooling to assess their implementation choices  🙂

Reply:
Not sure I understand. Could you give a concrete example of what parameters you would like to give (test files?), and what results you'd expect?

FI, the test runner already gives you the total time use for tests:

![image](https://user-images.githubusercontent.com/11834997/40951103-9fcb3416-6843-11e8-84fe-0feb450c237c.png)

Reply:
Sorry, you're right, I reacted from the notifications and did not have a clear vision of what this changeset brings in terms of functionality. What I meant was: if there is tooling to measure performance impact of changes, I'd love to have it brought to Core.

Looking at the diff, I indeed don't think this belongs to Core, due to the dependency to France but also to the lack of any machine-consumable report.

I would see the point of a performance metrics creation tool to follow the impact of changes in Core. As such though, without CI integration nor country-independent report creation, I don't see the point of sharing this in the codebase.

Reply:
OK so two options I see are to  use the `country-template` and to put this script in `openfisca-france`.
I like the second option because right now, this is were the issues are. 
@MattiSG about the machine consumable perf testing, @sandra found the circle API that shares some interesting meta-data on testing. my first inverstigations on the subject are [here](https://github.com/openfisca/openfisca-france/issues/1002)

Reply:
I'd also support the 2nd option :)